src/shared/utils/get-custom-tags.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const getCustomTags = async (domain:string) => {
+
+    //const url = `https://mediajel-tracker-custom-tags-staging.s3.amazonaws.com/`;
+    const url = `https://d3wuj95q2emo9.cloudfront.net/${domain}.js`
```

Comment:
we can swap `domain` with `window.location.hostname` This will give us the hostname of the site the tag will be installed on

So lets say the tag is installed in `https://www.budhut.net/deals` The `window.location.hostname` value would be `www.budhut.net` 

Then we should add a custom JS file in the S3 bucket that's referenced by the cloudfront url like `https://d3wuj95q2emo9.cloudfront.net/www.budhut.net.js`

Reply:
removed parameter "domain", used embedded window.location.hostname and have it encoded